core/maven-example/pom.xml

```diff
@@ -45,8 +46,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
 
         <configuration>
-          <source>1.5</source>
-          <target>1.5</target>
+          <source>1.7</source>
```

Comment:
minimum target of java ee 7

Reply:
works for us

Reply:
I think I remember a discussion somewhere: is stapler stil in-use outside Jenkins? If not, and even if so, why not go to Java 8? It's probably time anyway, as now JDK9 just got GA a few days ago, even JDK8 is legacy now ;-).

Reply:
It is in a few places (2 or 3 iirc)  7 is required for the servlet spec which is the min requirement here

Reply:
Stapler is on Java 8 already.